static/js/containers/UserPage_test.js

```diff
@@ -666,7 +666,9 @@ describe("UserPage", function() {
         return renderComponent(`/learner/${username}`, userActions).then(([, div]) => {
           let updatedProfile = _.cloneDeep(USER_PROFILE_RESPONSE);
           updatedProfile.username = SETTINGS.user.username;
-          updatedProfile.work_history.splice(0,1);
+          // the second entry will be the first one display in the UI
+          // (work history entries are sorted by date before they are rendered)
+          updatedProfile.work_history.splice(1,1);
```

Comment:
I suppose we could, I could try to get the company name from the UI and look for that in the entries.